portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase

```diff
@@ -0,0 +1,479 @@
+@component-name = "portal-questions"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Questions";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Questions Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			column = "2",
+			groupName = "Guest",
+			layoutName = "Questions Page",
+			widgetName = "Questions");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Message Boards");
+
+			if (IsElementPresent(locator1 = "Questions#QUESTIONS_LIST")) {
+				Questions.tearDown();
+			}
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "This is a test for LPS-159138. Verify if the user is able to subscribe a tag if they want to."
+	@priority = "5"
+	test CanASubscribeAQuestion {
+		task ("Create a Topic") {
+			JSONMBMessage.addCategory(
+				categoryDescription = "Description",
+				categoryName = "Questions Test Topic",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Create a Question") {
+			JSONQuestions.addQuestion(
+				bodyField = "Content AddQuestion",
+				categoryName = "Questions Test Topic",
+				groupName = "Guest",
+				titleField = "Question1");
+		}
+
+		task ("Go to My Subscription") {
+			Questions.gotoTab(questionTabField = "My Subscriptions");
+		}
+
+		task ("Validate the subcribed question") {
+			Questions.viewSubscribedQuestion(subscribedQuestion = "Question1");
+		}
+	}
+
+	@description = "This is a test for LPS-159144. Verify if the user is able to subscribe a tag if they want to."
+	@priority = "5"
+	test CanASubscribeATag {
+		task ("Create a Topic") {
+			JSONMBMessage.addCategory(
+				categoryDescription = "Description",
+				categoryName = "Questions Test Topic",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Create a Question with a tag") {
+			JSONQuestions.addQuestionWithTag(
+				bodyField = "Content1",
+				categoryName = "Questions Test Topic",
+				groupName = "Guest",
+				tagField = "new",
+				titleField = "Question1");
+		}
+
+		task ("Subscribe to Tag") {
+			Questions.openToTagsTab(
+				layoutName = "questions-page",
+				siteURLKey = "guest");
+
+			Questions.subscribeFromTag();
+		}
+
+		task ("Go to My Subscription page") {
+			Questions.gotoTab(questionTabField = "My Subscriptions");
+		}
+
+		task ("Validate the subcribed tag") {
+			Questions.viewSubscribedTag(subscribedTag = "new");
+		}
+	}
+
+	@description = "This is a test for LPS-159160. Verify if the user is able to subscribe a topic if they want to."
+	@priority = "5"
+	test CanASubscribeATopic {
```

Comment:
I would change the name from CanASubscribeAQuestion to CanSubscribeAQuestion, without the A from CanA